tensorflow/core/protobuf/rewriter_config.proto

```diff
@@ -107,10 +107,17 @@ message RewriterConfig {
   // Note that this can change the numerical stability of the graph and may
   // require the use of loss scaling to maintain model convergence.
   Toggle auto_mixed_precision = 23;
-  // Optimize data types for MKL (default is OFF).
+  // Optimize data types for oneDNN (default is OFF).
   // This will try to use bfloat16 on CPUs, which is faster.
   // Note that this can change the numerical stability of the graph.
-  Toggle auto_mixed_precision_mkl = 25;
+  // Note: this is deprecated.
+  // It is replaced by auto_mixed_precision_onednn_bfloat16
+  Toggle auto_mixed_precision_mkl = 31;
+  // Optimize data types for oneDNN (default is OFF).
+  // This will try to use bfloat16 on CPUs, which is faster.
+  // Note that this can change the numerical stability of the graph.
+  // Note: this is equivalent to the deprecated option auto_mixed_precision_mkl
+  Toggle auto_mixed_precision_onednn_bfloat16 = 25;
```

Comment:
We shouldn't change the assigned number. Just an FYI that I had changed `auto_mixed_precision_mkl` back to 25, and made `auto_mixed_precision_onednn_bfloat16` 31 internally before merging.

Reply:
ok, thanks and sorry about that.